.storybook/webpack.config.js

```diff
@@ -99,30 +83,6 @@ module.exports = (storybookBaseConfig, configType) => {
         },
       ],
     },
-    // For normal svg files (not icons) we should load the file normally
-    // and simply use it as a `<img src/>`.
-    {
-      test: function testForNormalSvgFiles(fileName) {
```

Comment:
we weren't actually bundling any "normal" SVG files, so removed this for readability.

Reply:
Nitpick: can we keep it there in case we need it in the future? I don't mind though to remove it...

Reply:
If you want I can re-add this, what I like about removing it is that it will make really apparent if someone adds SVG and PNG's because they will need to modify the webpack config.

Reply:
Sure, I don't mind. We can keep it removed for now